app/scripts/modules/titus/migration/titusMigrationConfig.component.ts

```diff
@@ -7,6 +8,11 @@ import './titusMigrationConfig.component.less';
 
 export interface ITitusMigrationViewState extends IViewState {
   accountsLoaded: boolean;
+  isDirty: boolean;
+  originalConfig: ITitusMigrationConfig;
+  originalStringVal: string;
+  saveError: boolean;
+  saving: boolean;
```

Comment:
Ok.  I removed and the compiler isn't complaining anymore.  Not sure what happened or what I saw originally.  Of course one can extend interfaces in TS so I shouldn't have needed to do that originally.  I removed them.